### PR TITLE
Modified the post added link to aem design

### PR DESCRIPTION
**Pull request by ragavan208**

### CODE REVIEW

File: _posts/2021-04-25-creating-akamai-flush-agent.md [hunk 1]
```diff
@@ -37,6 +37,10 @@ It is all automated, we don't need to clear the Akamai cache when new product go
 
 We don't ask DevOps to clear Akamai cache so customers will see latest page. Lets save their 1-2 mins time whenever new product launch.
 
+## Where you can find the code
+
+you can get it on GitHub [https://github.com/aem-design](https://github.com/aem-design).
```
Comment (wildone) [inline _posts/2021-04-25-creating-akamai-flush-agent.md]: maybe link to the component code?

File: _posts/2021-04-25-creating-akamai-flush-agent.md [hunk 2]
```diff
@@ -52,11 +56,13 @@ If you think you don't know these info's or not sure from where you can get it?
 
 After getting the information from you, we are encrypting data and storing it on AEM so you are safe with your secrets.
 
-## How to configure Akamai flush agent on your AEM Author instance
+## How to configure Akamai flush agent on your AEM author / publish instance
 
 By Assuming you have installed aem design code into your local AEM.
 
-Go to misc admin (http://localhost:4502/miscadmin) and open replication/agents.author
+Go to [miscadmin](http://localhost:4502/miscadmin)  and open replication/agents.author
```
Comment (wildone) [inline _posts/2021-04-25-creating-akamai-flush-agent.md]: spaces?

File: _posts/2021-04-25-creating-akamai-flush-agent.md [hunk 3]
```diff
@@ -52,11 +56,13 @@ If you think you don't know these info's or not sure from where you can get it?
 
 After getting the information from you, we are encrypting data and storing it on AEM so you are safe with your secrets.
 
-## How to configure Akamai flush agent on your AEM Author instance
+## How to configure Akamai flush agent on your AEM author / publish instance
```
Comment (wildone) [inline _posts/2021-04-25-creating-akamai-flush-agent.md:66]: can you do an intro on why author and why publisher?

**Comment by seeni-agl-zz** [comment]
hey Max can you have a look when you get time

**Comment by seeni-agl-zz** [comment]
Can you please have a look?

**Comment by seeni-agl-zz** [comment]
can you please have a look once you get time?

**Review by wildone — APPROVED** [review]

**Pull request merged**